codegen-winter/src/air/public_inputs.rs

```diff
@@ -0,0 +1,33 @@
+use ir::AirIR;
+
+use super::Scope;
+
+/// Updates the provided scope with a public inputs.
+pub(super) fn add_public_inputs_struct(scope: &mut Scope, ir: &AirIR) {
+    let name = "PublicInputs";
+    // define the custom Air struct.
```

Comment:
nit: this comment should probably say "PublicInputs" - same for the other comments in this file